src/components/layout/BaseLayout.tsx

```diff
@@ -2,20 +2,21 @@ import React, { FC, PropsWithChildren, ReactNode } from 'react';
 
 import { chakra, Container } from '@chakra-ui/react';
 
-import { sizes } from 'src/themes/shared/sizes';
+import { sizes } from 'src/themes/shared/tokens/sizes';
 
-import Flex from '../flex';
-import Divider from '../divider';
+import { Separator } from '../separator';
 
-interface Props {
+import { Flex } from '../flex';
+
+export interface BaseLayoutProps {
   header?: ReactNode;
   footer?: ReactNode;
   skyScraperAd?: ReactNode;
   heroAd?: ReactNode;
-  maxContentWidth: keyof typeof sizes.container;
+  maxContentWidth: keyof (typeof sizes)['container'];
```

Comment:
Out of curiosity - wouldn't:
```suggestion
  maxContentWidth: keyof typeof sizes.container;
```
still work here? It's a tad more readable without all the parentheses